nav2_msgs/action/FollowPath.action

```diff
@@ -1,10 +1,18 @@
+# Error codes
+int16 INVALID=0
+int16 UNKNOWN=1
+int16 FAILED_TO_MAKE_PROGRESS=2
+int16 PATIENCE_EXCEEDED=3
+int16 INVALID_PATH=4
```

Comment:
For discussion on exception codes:

These all make sense to me, but should there also be ones for:
- TF
- Could not generate
- Trajectories are in collision
- any others?

Also, `0` should be `NONE`, no?

Reply:
So, 

NoValidTrajectory (I can see a place to add that for dwb) -> Trajectories are in collision. 

I don't see a place for _could not generate trajectory_ in dwb. It seems like trajectory generation shouldn't cause any issues since it is just a projection. But maybe you have something in mind for MPPI? 

Fixed the message :). 

@doisyg are there any controller errors that you think should be added? 

```
int16 NONE=0
int16 UNKNOWN=1
int16 TF_ERROR=2
int16 NO_VALID_TRAJECTORIES=3
int16 FAILED_TO_MAKE_PROGRESS=4
int16 PATIENCE_EXCEEDED=5
int16 INVALID_PATH=6
```